tests/components/wyoming/test_satellite.py

```diff
@@ -472,7 +462,10 @@ async def on_muted(self):
             "homeassistant.components.wyoming.data.load_wyoming_info",
             return_value=SATELLITE_INFO,
         ),
-        patch("homeassistant.components.wyoming._make_satellite", make_muted_satellite),
+        patch(
+            "homeassistant.components.wyoming.switch.WyomingSatelliteMuteSwitch.async_get_last_state",
+            return_value=State("switch.test_mute", STATE_ON),
```

Comment:
This is normally not how restored state is tested. We have a fixture helper to help test it.